unit/cq_test.c

```diff
@@ -86,9 +124,10 @@ cq_open_close()
 
 		ret = create_cq(&cq, size, 0, FI_CQ_FORMAT_UNSPEC, FI_WAIT_UNSPEC);
 		if (ret == -FI_EINVAL) {
-			FT_WARN("\nSuccessfully completed %d iterations up to size %d "
-					"before the provider returned EINVAL...",
-					i + 1, size >> 1);
+			maximum_size = size >> 1;
+			FT_WARN(
+			    "\nSuccessfully completed %d iterations up to size %zu before the provider returned EINVAL...",
```

Comment:
print formatting options (%d, %zu) embedded in a string basically kill greppability.